raiden/tests/integration/test_integration_pfs.py

```diff
@@ -70,8 +77,12 @@ def test_pfs_send_capacity_updates_on_deposit_and_withdraw(
 
     # now we expect the room to be called the 1st time with a PFSCapacityUpdate
     # and a PFSFeeUpdate after the deposit
-    assert "PFSCapacityUpdate" in str(pfs_room.send_text.call_args_list[0])
-    assert "PFSFeeUpdate" in str(pfs_room.send_text.call_args_list[0])
+    assert "PFSCapacityUpdate" in str(pfs_rooms[0].send_text.call_args_list[0])
+    assert "PFSFeeUpdate" in str(pfs_rooms[0].send_text.call_args_list[0])
```

Comment:
This doesn't check the message themselves are correct, it would be better to decode the messages and compare them by value.

Reply:
Sure, I just mirrored what was there. But I'll update it

Reply:
I also propose postponing this to https://github.com/raiden-network/raiden/issues/5652